src/azul/service/responseobjects/step_function_helper.py

```diff
@@ -0,0 +1,59 @@
+import json
+
+from azul.deployment import aws
+
+
+class StepFunctionHelper:
+    """
+    Wrapper around boto3 SFN client to handle resource name generation and state machine executions
+    """
+    def state_machine_arn(self, state_machine_name):
+        return f'arn:aws:states:{aws.region_name}:{aws.account}:stateMachine:{state_machine_name}'
+
+    def execution_arn(self, state_machine_name, execution_name):
+        return f'arn:aws:states:{aws.region_name}:{aws.account}:execution:{state_machine_name}:{execution_name}'
+
+    def start_execution(self, state_machine_name, execution_name=None, execution_input=None):
+        """
```

Comment:
Remove this docstring except maybe for the reference to the boto3 method. This stuff is all obvious and can be looked up in the Boto3 documentation.